cmd/ketch/pool_remove.go

```diff
@@ -35,14 +36,28 @@ type poolRemoveOptions struct {
 }
 
 func poolRemove(ctx context.Context, cfg config, options poolRemoveOptions, out io.Writer) error {
-	pool := ketchv1.Pool{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: options.Name,
-		},
+	var pool ketchv1.Pool
+	var ns corev1.Namespace
+
+	if err := cfg.Client().Get(ctx, types.NamespacedName{Name: options.Name}, &pool); err != nil {
+		return fmt.Errorf("failed to get pool: %w", err)
 	}
+
+	// Get namespace that was created with the pool so it can be removed
+	if err := cfg.Client().Get(ctx, types.NamespacedName{Name: pool.Spec.NamespaceName}, &ns); err != nil {
+		return fmt.Errorf("failed to get namespace: %w", err)
+	}
+
 	if err := cfg.Client().Delete(ctx, &pool); err != nil {
 		return fmt.Errorf("failed to remove the pool: %w", err)
 	}
+
+	// Remove namespace after pool is deleted
+	if err := cfg.Client().Delete(ctx, &ns); err != nil {
```

Comment:
IMO we should prompt the user for confirmation to delete the namespace associated with the pool.  They may be using the namespace for something else.  I'd ask the user to confirm removal of the namespace by typing out the name of the namespace if they actually want to delete it.

Reply:
Good idea 👍

Reply:
agree, that's a good idea
just understand that I usually use "default" namespace for ketch pool